components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -50,7 +50,7 @@ class SchoolCard extends Component {
             cardFlipCallback={this.showFrontOfCard}
             locations={props.locations}
             logoSource={props.logoSource}
-            alt={`${props.schoolName} logo`}
+            alt={props.name}
```

Comment:
what if we just pass this as schoolName?

line 24 in backschoolcard would then be:
`<img src={logoSource} alt={`${schoolName} logo`} />`